Bottom: 5f3416087af062388709d700eb85cce4373a0a50
Top:    0ce0201361249fc89148a98a27ee5a6312f20568
Author: scott snyder <sss@karma>
Date:   2020-05-22 00:17:39 -0400

Fix compilation error.


---

diff --git a/sycl/include/CL/sycl/multi_ptr.hpp b/sycl/include/CL/sycl/multi_ptr.hpp
index c9873b93b60..9c5bef41bf2 100644
--- a/sycl/include/CL/sycl/multi_ptr.hpp
+++ b/sycl/include/CL/sycl/multi_ptr.hpp
@@ -516,11 +516,13 @@ public:
         static_cast<elem_pointer_t>(m_Pointer));
   }
 
+#if 0
   // Implicit conversion to multi_ptr<const void, Space>
   operator multi_ptr<const void, Space>() const {
     using ptr_t = typename detail::PtrValueType<const void, Space>::type *;
     return multi_ptr<const void, Space>(reinterpret_cast<ptr_t>(m_Pointer));
   }
+#endif
 
 private:
   pointer_t m_Pointer;
